actions/messagebird-send-sms/exo.action.yaml

```diff
@@ -0,0 +1,31 @@
+name: messagebird-send-sms
+description: Messagebird send sms
+handler: sms.php
+interpreter: php
+input:
+  type: object
+  required: [originator, recipients, text, key]
+  additionalProperties: false
+  properties:
+    originator:
+      type: string
+      description: originator
```

Comment:
User-facing text should be properly styled, capitalized, etc. In this case ~~originator~~ Originator. Same for ~~recipients~~ and ~~mesagebird key~~